.readme-partials.yml

```diff
@@ -80,20 +80,18 @@ body: |-
-  These trace fields, including [trace](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace), [spanId](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.span_id), and [traceSampled](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace_sampled) define the trace context for the log entry.
+  ## Automatic trace/span Id extraction
+  Cloud Logging libraries use [trace fields within LogEntry](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace) to capture trace contexts, which enables the [correlation of logs and traces](https://cloud.google.com/logging/docs/view/correlate-logs), and distributed tracing troubleshooting. 
+  These tracing fields including [trace](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace), [spanId](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.span_id), and [traceSampled](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.trace_sampled) define the trace context for a `LogEntry`.
 
-  If not provided explicitly, Google Cloud Logging library will automatically populate `LogEntry fields`_
-  `trace`, `span_id`, and `trace_sampled` from detected OpenTelemetry span context, or HTTP request headers.
+  If not provided explicitly in a LogEntry, Google Cloud Logging library will automatically populate `trace`, `span_id`, and `trace_sampled` fields from detected OpenTelemetry span contexts, or HTTP request headers.
 
-  ### Extracting from OpenTelemetry Context
+  ### Extracting trace/span Id from OpenTelemetry Context
   If you are using OpenTelemetry and there is an active span in the OpenTelemetry Context, that log entry will automatically have the `trace`, `span_id`, and `trace_sampled` fields populated from that span. More information about OpenTelemetry can be found [here](https://opentelemetry.io/docs/languages/js/).
 
-  ### Extracting from HTTP headers
-  If tracing fields are not provided explicitly and no OpenTelemetry context were detected, `trace` / `span_id` fields will be extracted automatically from HTTP headers. 
-  Trace information is automatically populated from either the `[W3C Traceparent](https://www.w3.org/TR/trace-context)` or `[X-Cloud-Trace-Context](https://cloud.google.com/trace/docs/trace-context#legacy-http-header)` headers.
-  Populating trace information this way also automatically populates the `http_request` field in the `LogEntry` as well.
+  ### Extracting trace/span Id from HTTP headers
+  If tracing fields are not provided explicitly and no OpenTelemetry context is detected, `trace` / `span_id` fields will be extracted automatically from HTTP headers. 
```

Comment:
detected, the `trace` ...  (add "the")

I'd also change "will be extracted" to "are extracted"